package.json

```diff
@@ -66,57 +66,58 @@
     "report-dir": "coverage-cypress"
   },
   "devDependencies": {
-    "@cypress/code-coverage": "^3.8.7",
-    "@cypress/webpack-preprocessor": "^5.5.0",
+    "@cypress/code-coverage": "^3.9.2",
+    "@cypress/webpack-preprocessor": "^5.6.0",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
     "@svgr/webpack": "^5.5.0",
-    "@testing-library/react-hooks": "^3.7.0",
+    "@testing-library/react-hooks": "^5.0.3",
     "@types/enzyme": "^3.10.8",
     "@types/enzyme-adapter-react-16": "^1.0.6",
```

Comment:
As you are aware, the `react-16` adapter isn't compatible with React 17, so we should either update the adapter to the unofficial `react-17` adapter (as you've done recently elsewhere) or stick with React 16 until other libraries catch up (as I've done in projects like CLUE).